src/mrinufft/operators/base.py

```diff
@@ -441,7 +441,9 @@ def compute_density(self, method=None):
             if `backend` is `tensorflow`, `gpunufft` or `torchkbnufft-cpu`
                 or `torchkbnufft-gpu`.
         """
-        if isinstance(method, np.ndarray):
+        if isinstance(method, np.ndarray) or (
+            CUPY_AVAILABLE and isinstance(method, cp.ndarray)
+        ):
```

Comment:
you have a conflict with master (due to my recent fix)

Reply:
Fixed it